app/views/people/edit.html.erb

```diff
@@ -1,6 +1,7 @@
 <% content_for :sidebar do %>
   <h1>Editing Person</h1>
-  <%= link_to 'Cancel', :back, class: 'btn btn-primary', style: 'width: 100%' %><br/><br/>
+  <%= sidebar_button_link 'Cancel', :back %>
```

Comment:
Yeah, I think I found this one in my next PR.  I'll see if I can grab that change into this one for the final version.